src/api/tests/test_order.py

```
@@ -1,41 +1,33 @@
+def test_init_buy_record_creates_order(app, monkeypatch):
+    from flaskr.service.order import funs as order_funs
 
-        record = init_buy_record(app, user_id, shifu.shifu_bid)
+    monkeypatch.setattr(order_funs, "get_shifu_creator_bid", lambda _app, _bid: "u1")
+    monkeypatch.setattr(order_funs, "set_shifu_context", lambda *_args, **_kwargs: None)
+    monkeypatch.setattr(
+        order_funs,
+        "get_shifu_info",
+        lambda _app, _bid, _preview: SimpleNamespace(price=Decimal("100.00")),
+    )
+    monkeypatch.setattr(order_funs, "query_and_join_active", lambda *args, **kwargs: [])
```

Ruff (0.14.13)

Unused lambda argument: `args`

(ARG005)